data-prep/README.md

```diff
@@ -15,9 +15,59 @@ Data Cleaning and Munging
 
 A number of cleaning steps are necessary to use the raw data.
 
-### Project Filtering
+### Consumption Data Cleaning
+
+**Elec Traces**
+
+* Remove duplicates
+  * No duplicates in monthly data
```

Comment:
Not sure we can make this comment since as far as I know nobody has tried to deduplicate the monthly billing data...